spec/registrant_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "spec_helper"
+require "./lib/registrant"
+
+describe "Registrant" do
+  before(:each) do
+    @registrant_1 = Registrant.new("Bruce", 18, true)
+    @registrant_2 = Registrant.new("Penny", 15)
+  end
+
+  describe "#initialize" do
+    it "exists" do
+      expect(@registrant_1).to be_a(Registrant)
+      expect(@registrant_2).to be_a(Registrant)
+    end
+
+    it "has a @name" do
+      expect(@registrant_1.name).to eq("Bruce")
+      expect(@registrant_2.name).to eq("Penny")
+    end
+
+    it "has an @age" do
+      expect(@registrant_1.age).to eq(18)
+      expect(@registrant_2.age).to eq(15)
+    end
+    
+    it "has @license_data as a Hash" do
+      expect(@registrant_1.license_data).to be_a(Hash)
```

Comment:
I don't think this test is necessary because you are the one setting the Hash in the code, so it will always be a Hash.